app/hub/Views/HomeView/HomeView.jsx

```diff
@@ -34,9 +34,21 @@ const HomeView = (props) => {
 		isPlus,
 	} = props;
 	const accountHref = `https://account.${globals.GHOSTERY_DOMAIN}.com`;
-	const headerInfoText = (globals.BROWSER_INFO && globals.BROWSER_INFO.name === 'firefox')
-		? t('hub_home_header_info_opted_out')
-		: t('hub_home_header_info');
+
+	let headerInfoText;
+	if (globals.BROWSER_INFO) {
+		switch (globals.BROWSER_INFO.name) {
+			case 'firefox':
+				headerInfoText = t('hub_home_header_info_opted_out');
+				break;
+			case 'cliqz':
+				headerInfoText = t('hub_home_header_info_cliqz');
+				break;
+			default:
+				headerInfoText = t('hub_home_header_info');
+		}
+	}
+
```

Comment:
This introduces a dependency on `globals.BROWSER_INFO` being set compared to the old code; let's refactor to avoid doing that. We can do

```
let headerInfoText = t('hub_home_header_info');
if (globals.BROWSER_INFO) {
    // switching to if/else to avoid duplication in the default case of switch
    if (globals.BROWSER_INFO.name === 'firefox') { headerInfoText = ....}
    else if (.... === 'cliqz') { .....}
}
```

Reply:
Great catch. Fixed.